spark-jobs/src/main/scala/za/co/absa/enceladus/standardization_conformance/config/StdConformanceConfig.scala

```diff
@@ -0,0 +1,119 @@
+  override def withDatasetName(value: String): StdConformanceConfig = copy(datasetName = value)
+  override def withDatasetVersion(value: Int): StdConformanceConfig = copy(datasetVersion = value)
+  override def withReportDate(value: String): StdConformanceConfig = copy(reportDate = value)
+  override def withReportVersion(value: Option[Int]): StdConformanceConfig = copy(reportVersion = value)
+  override def withPerformanceMetricsFile(value: Option[String]): StdConformanceConfig = copy(performanceMetricsFile = value)
+  override def withFolderPrefix(value: Option[String]): StdConformanceConfig = copy(folderPrefix = value)
+  override def withPersistStorageLevel(value: Option[StorageLevel]): StdConformanceConfig = copy(persistStorageLevel = value)
+
+  override def withRawFormat(value: String): StdConformanceConfig = copy(rawFormat = value)
+  override def withCharset(value: Option[String]): StdConformanceConfig = copy(charset = value)
+  override def withRowTag(value: Option[String]): StdConformanceConfig = copy(rowTag = value)
+  override def withCsvDelimiter(value: Option[String]): StdConformanceConfig = copy(csvDelimiter = value)
+  override def withCsvHeader(value: Option[Boolean]): StdConformanceConfig = copy(csvHeader = value)
+  override def withCsvQuote(value: Option[String]): StdConformanceConfig = copy(csvQuote = value)
+  override def withCsvEscape(value: Option[String]): StdConformanceConfig = copy(csvEscape = value)
+  override def withCobolOptions(value: Option[CobolOptions]): StdConformanceConfig = copy(cobolOptions = value)
+  override def withFixedWidthTrimValues(value: Option[Boolean]): StdConformanceConfig = copy(fixedWidthTrimValues = value)
+  override def withRawPathOverride(value: Option[String]): StdConformanceConfig = copy(rawPathOverride = value)
+  override def withFailOnInputNotPerSchema(value: Boolean): StdConformanceConfig = copy(failOnInputNotPerSchema = value)
+
+  override def withCredsFile(value: Option[String], menasCredentialsFactory: MenasCredentialsFactory): StdConformanceConfig = {
+    copy(credsFile = value, menasCredentialsFactory = menasCredentialsFactory)
+  }
+  override def withAuthKeytab(value: Option[String], menasCredentialsFactory: MenasCredentialsFactory): StdConformanceConfig = {
+    copy(keytabFile = value, menasCredentialsFactory = menasCredentialsFactory)
+  }
+}
+
+object StdConformanceConfig {
+
+  def getCmdLineArguments(args: Array[String]): StdConformanceConfig = {
+    //Mutually exclusive parameters don't work in scopt 4
+    if (args.contains("--menas-credentials-file") && args.contains("--menas-auth-keytab")) {
+      println("ERROR: Only one authentication method is allowed at a time")
+      System.exit(1)
+    }
+    val optionCmd = OParser.parse(stdConfJobParser, args, StdConformanceConfig())
+
+    if (optionCmd.isEmpty) {
+      // Wrong arguments provided, the message is already displayed
+      System.exit(1)
```

Comment:
Could we refactor a bit and get rid of these system exists?

Reply:
Yea, it was leftover from before, I will remove it